x-pack/plugins/session_view/public/components/SessionViewDetailPanel/styles.ts

```diff
@@ -25,7 +25,7 @@ export const useStyles = ({ height = 500 }: StylesDeps) => {
         right: '0',
       },
       to: {
-        right: '-100%',
+        right: '-424px',
```

Comment:
should we be hard coding this width?

Reply:
it was causing a small flickering bug in the animation,

since the component is mounted together with the animation, the browser takes some time to translate the width of 100% into 424px. if we change the animation time from 200ms to 2000ms the bug is more prominent

Reply:
ah I see so it's trigger the animation too early? maybe we can use `useLayoutEffect`? Since that waits until the DOM is ready so in theory the component should have a width

Reply:
yeah, need to test but I think `useLayoutEffect` should work!

Reply:
Was talking to Paulo about the cause of delay and it seems like it's because the detail panel is initially at the very right of the split panel with the `right: 0;`. so it took some more time to slide to the left 😆 

![image](https://user-images.githubusercontent.com/16872649/144308094-1af144d0-36d8-4324-9b70-fca6787b7ece.png)

An option we discussed is to use `transform` instead of `right`, but eui inner panel has the `transform: none !important;` for split panel. We will need to use something other than the split panel component if we want to do that.

I think the easiest way is to just use pixels. I don't think it will hurt us in this case. What are everyone's thoughts?